### PR TITLE
MM-17620 - Web: Error when entering an invalid MFA token is unclear

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
When an invalid format is used for an MFA token, (ie, Alpha characters or not 6/8 digits). The Authenticate method returns an error which we handle differently. We can't change the error id, as the web app code handles these error differently. Therefore we need to simply change the error text to match the other error (valid formatted, but invalid token(.
```{
    "id": "api.user.check_user_mfa.bad_code.app_error",
    "translation": "Invalid MFA token."
}
```

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17620

### CODE REVIEW

**Comment by hanzei** [comment]
@sbishel Is this ready for review?

**Comment by sbishel** [comment]
@hanzei - Yes, do I need to set something?

**Comment by hanzei** [comment]
See https://github.com/mattermost/mattermost-webapp/pull/3682#issuecomment-532240448

File: i18n/en.json [hunk 1]
```diff
@@ -4176,7 +4176,7 @@
   },
   {
     "id": "mfa.validate_token.authenticate.app_error",
-    "translation": "Error trying to authenticate MFA token"
```
Comment (wiersgallak) [inline i18n/en.json]: @sbishel Wondering if this can be simplified to "Error authenticating MFA token"

Comment (sbishel) [reply i18n/en.json]: That is the old wording, I matched the new wording to when the code is bad. 
```{
    "id": "api.user.check_user_mfa.bad_code.app_error",
    "translation": "Invalid MFA token."
  },```

Comment (wiersgallak) [reply i18n/en.json]: My bad - this is good to go then from my perspective.  Thanks for clarifying

**Pull request merged**